src/app/features/tech-record/create-batch-trl/components/batch-trl-details/batch-trl-details.component.ts

```diff
@@ -0,0 +1,118 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { AbstractControl, FormArray, FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { GlobalError } from '@core/components/global-error/global-error.interface';
+import { GlobalErrorService } from '@core/components/global-error/global-error.service';
+import { DynamicFormService } from '@forms/services/dynamic-form.service';
+import { FormNodeWidth } from '@forms/services/dynamic-form.types';
+import { CustomValidators } from '@forms/validators/custom-validators';
+import { TechnicalRecordService } from '@services/technical-record/technical-record.service';
+import { combineLatest, Observable, Subject, take } from 'rxjs';
+
+@Component({
+  selector: 'app-batch-trl-details',
+  templateUrl: './batch-trl-details.component.html',
+  styleUrls: ['./batch-trl-details.component.scss']
+})
+export class BatchTrlDetailsComponent implements OnInit, OnDestroy {
+  form: FormGroup;
+
+  private startingObject = { vin: '' };
```

Comment:
There isn't really much of a point to this starting object

Reply:
Removed.